src/main/java/TesterVisitor.java

```diff
@@ -101,9 +107,81 @@ public Value visitVarReassignment(TesterParser.VarReassignmentContext ctx) {
 
     @Override
     public Value visitRequest(TesterParser.RequestContext ctx) {
-        return super.visitRequest(ctx);
+
+        String method = ctx.method().getText();
+        String endpoint = processString(ctx.endpoint().getText());
+        String jsonBody = ctx.obj() != null ? parseJsonObj(ctx.obj()) : null;
+
+        HttpResult response;
+        System.out.println("-------------------------------------");
+        System.out.println(currentTestCaseName + ":");
+        Duration duration = Duration.ofMillis(currentTestTimeout <= 0 ? DEFAULT_TIMEOUT_MS : currentTestTimeout);
+        if (currentTestRepeats > 1 ) {
+            switch (method) {
+                case "GET" -> response = executor.sendRequestBenchmark(endpoint, currentTestRepeats, duration,"GET");
+                case "HEAD" -> response = executor.sendRequestBenchmark(endpoint, currentTestRepeats, duration,"HEAD");
+                case "DELETE" -> response = executor.sendRequestBenchmark(endpoint, currentTestRepeats, duration,"DELETE");
+                case "POST" -> response = executor.sendRequestBenchmark(endpoint, currentTestRepeats, duration, jsonBody,"POST");
+                case "PUT" -> response = executor.sendRequestBenchmark(endpoint, currentTestRepeats, duration, jsonBody,"PUT");
+                default -> throw new RuntimeException("Unsupported method: " + method);
+            }
+        } else {
+            switch (method) {
+                case "GET" -> response = executor.sendGet(endpoint, duration);
+                case "HEAD" -> response = executor.sendHead(endpoint, duration);
+                case "DELETE" -> response = executor.sendDelete(endpoint, duration);
+                case "PUT" -> response = executor.sendPut(endpoint, jsonBody, duration);
+                case "POST" -> response = executor.sendPost(endpoint, jsonBody, duration);
+                default -> throw new RuntimeException("Unsupported method: " + method);
+            }
+        }
+
+
+        System.out.println("Send " + method + " " + endpoint);
+        System.out.println(response);
+        System.out.println("-------------------------------------");
+
+
+        currentTestRepeats = 0;
+        currentTestTimeout = 0;
+
+        return null;
     }
 
+    private String parseJsonObj(TesterParser.ObjContext ctx) {
+        if (ctx.pair().isEmpty()) return "{}";
+
+        StringBuilder json = new StringBuilder("{");
+        List<TesterParser.PairContext> pairs = ctx.pair();
+        for (int i = 0; i < pairs.size(); i++) {
+            var pair = pairs.get(i);
+            String key = stripQuotes(pair.STRING().getText());
```

Comment:
```suggestion
            String key = processString(pair.STRING().getText());
```
This might be redundant, but it doesn’t hurt to have it.